binary/proto/scan_result.proto

```diff
@@ -988,6 +989,10 @@ message SecretData {
     string access_id = 1;
     string secret = 2;
   }
+
+  message ReCaptchaKey{
+    string secret = 1;
```

Comment:
Is the public site key also needed for being able to authenticate to the service? If so then we should also try to detect and store that